doc/content/reference/gateway-rtt/_index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Gateway RTT"
+description: "Gateway RTT"
+---
+
+{{% tts %}} records Round Trip Times (RTT) between the Gateway Server and Gateways which act as a measure of latency.
+
+This section explains how {{% tts %}} records and uses these values.
+
+<!--more-->
+
+## Background
+
+Since {{% tts %}} communicates with gateways over the internet, a latency of transmitted messages is expected. This latency is not an issue for uplinks, but for downlinks, since the Rx windows are at specific intervals, the {{% tts %}} needs to be aware of network latency.
+
+For UDP gateways, the RTT is the time difference between a downlink message and the corresponding valid Tx Acknowledgement message.
```

Comment:
I'm wondering if it might be useful to stress the specific moment this refers to, for example time difference from the moment a downlink message was **sent** and the moment when a corresponding valid Tx ACK was **received**?